test/framework/scripts.py

```diff
@@ -46,7 +46,7 @@ def test_generate_software_list(self):
         # adjust $PYTHONPATH such that test easyblocks are found by the script
         eb_blocks_path = os.path.abspath(os.path.join(os.path.dirname(__file__), 'sandbox'))
         pythonpath = os.environ.get('PYTHONPATH', '.')
-        os.environ['PYTHONPATH'] = "%s:%s" % (pythonpath, eb_blocks_path)
+        os.environ['PYTHONPATH'] = ".:%s:%s" % (pythonpath, eb_blocks_path)
```

Comment:
my bad, I was fiddling with my `$PYTHONPATH`, I don't think this should be her (at least not like this)